app/services/sign_in/auth_sso/session_validator.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module SignIn
+  module AuthSSO
+    class SessionValidator
+      def initialize(access_token:, client_id:)
+        @access_token = access_token
+        @client_id = client_id
+      end
+
+      def perform
+        validate_session!
+        validate_client_configs!
+        validate_credential_level_and_type!
+
+        auth_sso_user_attributes
+      end
+
+      private
+
+      attr_reader :access_token, :client_id
+
+      def auth_sso_user_attributes
+        {
+          idme_uuid: user_verification.idme_uuid || user_verification.backing_idme_uuid,
```

Comment:
Is it possible for any of these objects to be nil? Just thinking of potentially adding in a safe navigation operator to avoid any unexpected `NoMethodError` errors

Reply:
good question. we are good since before this we validate that a `session` exists and a `session` must have a `user_verification`